modules/ROOT/pages/manage-index.adoc

```diff
@@ -7,6 +7,7 @@ After installing and configuring Runtime Fabric and deploying Mule applications
 * Use Anypoint Monitoring and Visualizer to monitor applications deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitoring Applications Deployed to Runtime Fabric].
 * Install rtfctl to manage Runtime Fabric settings. See xref:install-rtfctl.adoc[Install rtfctl].
 * Troubleshoot errors with installation or post-installation. See xref:troubleshoot-guide.adoc[Troubleshooting Runtime Fabric].
+* Stop and restart a Runtime Fabric. This requires stopping and restarting each instance.
```

Comment:
why is there an article here? We don't usually put articles before product/feature names.

Reply:
Runtime Fabric is often referred to as an 'environment', and in this particular context, there is an article preceding it. There are various places in the docs where this occurs.